js/data.js

```diff
@@ -1,8 +1,19 @@
 /* exported data */
-
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+var previousData = localStorage.getItem('code-journal-entries');
+
+if (previousData !== null) {
+  data.entries = JSON.parse(previousData);
```

Comment:
Saving the full data object and placing it as the value of the entries property? Did you double-check that your app still worked before resubmitting? After one refresh, this should break the entire app.